Makefile

```diff
@@ -278,90 +299,187 @@ endif
 ifeq ($(SR_FORTRAN),OFF)
 SKIP_FORTRAN = --ignore ./tests/fortran
 endif
+SKIP_DOCKER := --ignore ./tests/docker
+
+# Build SSDB string for clustered database
+SSDB_STRING := 127.0.0.1:$(SR_TEST_PORT)
+PORT_RANGE := $(shell seq `expr $(SR_TEST_PORT) + 1` 1 `expr $(SR_TEST_PORT) + $(SR_TEST_NODES) - 1`)
+SSDB_STRING += $(foreach P,$(PORT_RANGE),",127.0.0.1:$(P)")
+SSDB_STRING := $(shell echo $(SSDB_STRING) | tr -d " ")
+
+# Run test cases with a freshly instantiated standalone Redis server
+# Parameters:
+# 	1: the test directory in which to run tests
+define run_smartredis_tests_with_standalone_server
+	echo "Launching standalone Redis server" && \
+	export SR_TEST_DEVICE=$(SR_TEST_DEVICE) SR_SERVER_MODE=Standalone && \
+	export SMARTREDIS_TEST_CLUSTER=False SMARTREDIS_TEST_DEVICE=$(SR_TEST_DEVICE) && \
+	export SSDB=127.0.0.1:$(SR_TEST_PORT) && \
+	python utils/launch_redis.py --port $(SR_TEST_PORT) --nodes 1 \
+		--rai $(SR_TEST_REDISAI_VER) --device $(SR_TEST_DEVICE) && \
+	echo "Running standalone tests" && \
+	PYTHONFAULTHANDLER=1 python -m pytest $(SR_TEST_PYTEST_FLAGS) $(COV_FLAGS) \
```

Comment:
Incredibly obscure corner case that I'm not sure is a problem or that we even want to handle, but thought I would bring to your attention just in case:

If a new dev, in a brand new python env, runs `make test` but does not set `SR_PYTHON=ON`, then `pip install -e .[dev]` will never get run, and thus `pytest` will never be installed, and thus this command will error out here and leave the a redis db running.

(like I said, very obscure, not even sure its an issue, hahaha)

Reply:
A similar error can occur for the clustered/uds version of this fn

Reply:
It's a potential issue for sure, but not one introduced with this PR. The only real way to address it would be to put a call to pip install into the Makefile, and as a group we decided against that with the build system refactor PR